docs/release-notes/july-2023.md

```diff
@@ -0,0 +1,231 @@
+---
+slug: july-2023
+append_help_link: true
+title: July 2023
+hide_title: true
+description: >-
+  Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+---
+
+# July 2023 release notes
+
+## Semgrep OSS Engine
+
+This section of release notes includes upgrades of Semgrep OSS Engine for versions ranging between **1.31.0** and **1.34.1**.
+
+### Added
+
+* Added rule option `interfile: true`, which is set under the `options` key. This rule option replaces setting interfile under the `metadata` key. 
```

Comment:
If I were an external user I'd be worried that "replaces" means "the previous option is no longer supported", which AFAIK is not the case. What about "The rule-level option is now preferred to setting interfile..."